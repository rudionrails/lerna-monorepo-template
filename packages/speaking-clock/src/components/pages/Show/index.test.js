import React from "react";

// module under test
import Index from "./index";

test("to match snapshot", () => {
  const subject = shallow(<Index />);
  expect(subject).toMatchSnapshot();
});
