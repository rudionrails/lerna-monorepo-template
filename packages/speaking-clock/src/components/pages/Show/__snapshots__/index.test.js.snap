// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`to match snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
